### DISCUSSION TITLE
PR のレビュー依頼

### DISCUSSION THREAD

**Discussion by C-Naoki** [increments/qiita-discussions#910] [要望・アイデア・不具合]
いつも Qiita を利用させてもらっています。qiita-cli によって、記事を github で管理できるようになり、より快適になっております。

今回の要望は、単刀直入に申し上げますと、[PR](https://github.com/increments/qiita-cli/pull/187) を出したので、反映していただきたいというものです。実装した内容としては、github workflow にて自動的に push が走った時の、commit message をユーザー側で設定できなかったので、それを新たな変数 `commit-message` を設けることで、設定できるようにいたしました。

よろしければ、レビュー及びマージの方をしていただけますと、幸いです。よろしくお願いいたします。

**Comment by tanaka-asami415** [comment]
@C-Naoki 
喜んでいただけて大変嬉しいです！

>[PR](https://github.com/increments/qiita-cli/pull/187) を出したので、反映していただきたいというものです。

ありがとうございます！レビューさせていただきます🙇

**Reply by P-SiZK** [reply]
@C-Naoki 
変更を確認して Approve しました！
既にリリースを行い、`v1.6.0` としてお使いいただけます！

https://github.com/increments/qiita-cli/releases/tag/v1.6.0

Pull Request を送っていただきありがとうございました :bow:

**Reply by C-Naoki** [reply]
こちらこそ、迅速な対応ありがとうございました！大変助かりました！
